server/service/src/programs/patient/update.rs

```diff
@@ -78,6 +73,15 @@ pub fn update_patient(
                     DocumentInsertError::InvalidParent(_) => UpdatePatientError::InvalidParentId,
                 })?;
 
+            if is_latest_doc(ctx, service_provider, &doc)
+                .map_err(UpdatePatientError::DatabaseError)?
+            {
+                // update the names table
+                patient_document_updated(&ctx.connection, &doc_timestamp, patient)?;
```

Comment:
Nit: Rename function? The patient table is being updated, not the patient's document?

Reply:
I used it in a "callback" style, i.e. the caller just says: "the patient document has been updated; now do what every is needed". Its like sending an event. However, think I will rename it to `update_patient_name()` anyway...